Tests/AppTests/Common/Extensions/HTTP/Responder/Responder+Authorization.swift

```diff
@@ -5,18 +5,24 @@ extension Responder {
   public func unauthorizedTestResponse(
     to method: HTTP.Method,
     at path: String,
+    query: String? = nil,
     hostname: String = "0.0.0.0",
     headers: [HTTP.HeaderKey: String] = [:],
     body: BodyRepresentable? = nil,
     file: StaticString = #file,
     line: UInt = #line
     ) throws -> HTTP.Response {
-    return try self.testResponse(
-      to: method,
-      at: path,
-      hostname: hostname,
+    let req = Request.makeTest(
+      method: method,
       headers: headers,
-      body: body,
+      body: body?.makeBody() ?? .data([]),
```

Comment:
what a magic ?

Reply:
if function have body passed in parameter we create it, else we create empty data :)